OCPBUGS-30865: Improve OpenStack CCM config reference for dual-stack (#73031)
openshift/openshift-docs: modules/cluster-cloud-controller-config-osp.adoc

| Optional. The load balancer API version. Only `"v2"` is supported.

| `subnet-id`
| The ID of the Networking service subnet on which load balancer VIPs are created. In dual-stack deployments it's recommended to leave this option unset and let cloud-provider-openstack autodetect which subnet to use for a load balancer.

// This ID is also used to create pool members if `member-subnet-id` is not set.

// | `member-subnet-id`
// | ID of the Neutron network on which to create the members of the load balancer. The load balancer gets another network port on this subnet. Defaults to `subnet-id` if not set.

| `network-id`

> | The ID of the Networking service network on which load balancer VIPs are created. Unnecessary if `subnet-id` is set. If not set network will be autodetected based on the network used by cluster nodes.

Reviewer: ```suggestion
| The ID of the Networking service network on which load balancer VIPs are created. Unnecessary if `subnet-id` is set. If this property is not set, the network is automatically selected based on the network that cluster nodes use.
```
